src/test/java/test/ReflectionHelperTest.java

```diff
@@ -0,0 +1,64 @@
+package test;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import org.testng.collections.Lists;
+import org.testng.internal.reflect.ReflectionHelper;
+import test.github765.DuplicateCallsSample;
+
+import java.lang.reflect.Method;
+import java.util.List;
+
+public class ReflectionHelperTest {
+
+    @Test
+    public void testMethodCount() {
```

Comment:
Ok! I don't know if it has sense but could you just add a test case with bridge methods without inheritance?